src/components/AssignmentFeedbackForm/AssignmentFeedbackForm.tsx

```diff
@@ -54,59 +66,106 @@ const AssignmentFeedbackForm = () => {
   };
 
   return (
-    <form onSubmit={form.onSubmit((values) => onSubmit(values))}>
-      <Text>
-        Please complete this short questionaire. Once you&apos;re done we will
-        let you download your answers so you can submit to coderunner.
+    <Stack>
+      <Title order={2}>Please rate your experience</Title>
+      <Text size='xl'>
+        Please complete this <b>optional</b>, short questionaire. Your answers
+        are below the questionaire.
         <br />
-        Important: you must submit to coderunner to get your marks for this
-        course!
+        Important: you must submit your answers to coderunner to get your marks
+        for this course!
       </Text>
-      <InputWrapper label='How easy was it to use this tool?' required>
-        <Slider
-          label={(val) => easyMarks?.find((mark) => mark.value === val)?.label}
-          step={5}
-          marks={easyMarks}
+      <form
+        onSubmit={form.onSubmit((values) => onSubmit(values))}
+        className='flex flex-col gap-y-8'
+      >
+        <InputWrapper
+          className='mb-4'
+          label='How easy was it to use this tool?'
+          classNames={{ label: 'text-lg' }}
+          required
+        >
+          <Group className='justify-between'>
+            <Text size='sm'>Extremely Difficult</Text>
+            <Text size='sm'>Extremely Easy</Text>
+          </Group>
+          <Slider
+            label={(val) => marks?.find((mark) => mark.value === val)?.label}
+            size='lg'
+            step={5}
+            marks={marks}
+            classNames={{
+              bar: 'bg-blue-600',
+              markFilled: 'border-blue-600',
+              thumb: 'border-blue-600',
+              markLabel: 'font-bold text-gray-800',
+            }}
+            // eslint-disable-next-line react/jsx-props-no-spreading
```

Comment:
You've already turned this off in the rules so I don't believe this is necessary anymore?

Reply:
dang i forgot sorry